lib/cog/chat/slack/connector.ex

```diff
@@ -323,6 +332,44 @@ defmodule Cog.Chat.Slack.Connector do
     end
   end
 
+  defp normalize_channel_id(:by_id, id = "C" <> _, slack) do
```

Comment:
Oh yeah, good point. I just added the `"C"` bit as an extra check. I can drop it if you think it's confusing.